src/main/java/frc/team2412/robot/commands/diagnostic/LauncherDiagnosticCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.team2412.robot.commands.diagnostic;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.commands.launcher.SetAngleCommand;
+import frc.team2412.robot.commands.launcher.SetLaunchSpeedCommand;
+import frc.team2412.robot.commands.launcher.StopLauncherCommand;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class LauncherDiagnosticCommand extends SequentialCommandGroup {
+	private final LauncherSubsystem launcherSubsystem;
+	private final double Angle;
+
+	public LauncherDiagnosticCommand(LauncherSubsystem launcherSubsystem) {
+		this.launcherSubsystem = launcherSubsystem;
+		this.Angle = launcherSubsystem.getAngle();
+		addCommands(
+				new SetAngleCommand(launcherSubsystem, 45),
+				new SetAngleCommand(launcherSubsystem, Angle),
+				new SetLaunchSpeedCommand(launcherSubsystem, 100),
```

Comment:
`SetLaunchSpeedCommand` finishes as soon as it starts, so likely need a wait after this command in order to see the launcher speed up and verify it reaches the expected speed value.

Reply:
made it so it uses .withTimeout!